pkg/nmi/standard.go

```diff
@@ -51,6 +51,7 @@ func (sc *StandardClient) GetIdentities(ctx context.Context, podns, podname, cli
 	// filter out if we are in namespaced mode
 	var filterPodIdentities []aadpodid.AzureIdentity
 	for _, val := range podIDs {
+		val := val
```

Comment:
Let's add a comment for these changes/update the PR description on why this is done